BO4E/ENUM/FernsteuerbarkeitStatus.cs

```diff
@@ -0,0 +1,37 @@
+using System.Runtime.Serialization;
+
+using ProtoBuf;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Status der Fernsteuerbarkeit einer Marktlokation gemäß den Vorgaben für die Direktvermarktung.
+    /// </summary>
+    [ProtoContract]
+    public enum FernsteuerbarkeitStatus
+    {
+        /// <summary>
+        /// Marktlokation ist technisch nicht fernsteuerbar.
+        /// Der NB bestätigt mit der Anmeldung einer erzeugenden Marktlokation zur Direktvermarktung, dass die Anlage nicht mit einer Fernsteuerung ausgestattet ist und nicht fernsteuerbar ist. Die Voraussetzung zur Zahlung der Managementprämie für fernsteuerbare Anlagen ist nicht gegeben.
+        /// </summary>
+        [ProtoEnum(Name = nameof(FernsteuerbarkeitStatus) + "_" + nameof(Z96_NICHT_FERNSTEUERBAR))]
+        [EnumMember(Value = "Z96_NICHT_FERNSTEUERBAR")]
+        Z96_NICHT_FERNSTEUERBAR,
```

Comment:
mach mal bitte bei den enums nicht den EDIFACT-Code mit rein, also ohne Z96 in dem Fall